components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -206,15 +210,30 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
         if (channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) {
             leaveChannelText = intl.formatMessage({id: 'sidebar_left.sidebar_channel_menu.leaveConversation', defaultMessage: 'Close Conversation'});
         }
+        const color = () => {
+            if (channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) {
+                return 'rgba(var(--center-channel-color-rgb), 0.72)';
+            } else if (!(channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) && !this.state.closeIconHovered) {
+                return 'var(--error-text)';
+            }
+            return 'white';
+        };
 
         let leaveChannel;
         if (channel.name !== Constants.DEFAULT_CHANNEL) {
             leaveChannel = (
                 <Menu.Group>
                     <Menu.ItemAction
+                        onMouseEnter={() => this.setState({closeIconHovered: true})}
+                        onMouseLeave={() => this.setState({closeIconHovered: false})}
                         id={`leave-${channel.id}`}
                         onClick={this.handleLeaveChannel}
-                        icon={<i className='icon-close'/>}
+                        icon={
+                            <CloseIcon
+                                size={16}
+                                color={color()}
```

Comment:
sry, but this adds a ton of complexity ... my fault... I just had a look at how the icon gets rendered inside the `MenuItem` component and it always wraps it in a `<span className='icon'>` component. We will postpone this once we run a campaign to replace all of the `i` tags with `svg` icon components.

If you could just revert your latest changes that will be fine for now.

Again I am sorry that I requested that change. I thought it is easier to implement, but apparently needs more work than anticipated.